src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -55,6 +60,7 @@ public static int Run(string[] args)
                 publish.OutputPath = outputOption.Value();
                 publish.Configuration = configurationOption.Value();
                 publish.VersionSuffix = versionSuffixOption.Value();
+                publish.ProfileProj = profileProjOption.Value();  //TODO: Merge multiple profile files
```

Comment:
If you are not going to do this now, I would prefer having an issue tracking this rather than a TODO in the code.